pcsx2/x86/iR3000A.cpp

```diff
@@ -941,13 +941,13 @@ static void iPsxBranchTest(u32 newpc, u32 cpuBranch)
 		xADD(eax, blockCycles);
 		xMOV(ptr32[&psxRegs.cycle], eax); // update cycles
 
-		if ((psxHu32(HW_ICFG) & (1 << 3))) // if were in PS1 mode adjust ee accordingly.
+		if ((psxHu32(HW_ICFG) & (1 << 3))) // if we're in PS1 mode adjust ee accordingly.
 		{
 			psxEEmultiplier = 8.71;
 		}
 
 		// jump if iopCycleEE <= 0  (iop's timeslice timed out, so time to return control to the EE)
-		xSUB(ptr32[&iopCycleEE], blockCycles*static_cast<int>(psxEEmultiplier));
+		xSUB(ptr32[&iopCycleEE], (s32)(round(blockCycles* psxEEmultiplier)));
```

Comment:
does psxEEmultiplier default to 8? if it's uninitialised (as it looks to only be set above) it's going to crash when in PS2 mode.

Reply:
Yeah, I'd like it to be set to 8 in psxReset() also, so if they start a new game (a PS2 one) the multiplier isn't off.